app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Public Recipes</title>
+  <%= stylesheet_link_tag 'public_recipe_index', media: 'all', 'data-turbolinks-track': 'reload' %>
+</head>
+<body>
+  <div class="container">
+    <h2>Public recipes</h2>
+
+
+  <div class= "public-recipe">
+    <% if @public_recipes.empty? %>
+      <p>No public recipes added yet</p>
+    <% else %>
+      <% @public_recipes.each do |recipe| %>
+        <div>
+          <h3><%= link_to recipe.name, user_recipe_path(user_id: recipe.user_id, id: recipe.id), class: "recipe-name" %></h3>
```

Comment:
- [ ] In the public recipes, there will be many user's recipes. As per the requirement, if the recipe belongs to the author, there should be a remove button. Kindly implement this.

Reply:
@tugsem 

I think you got confused. in the public recipes there's no delete button. 

![image](https://github.com/smunoz1988/recipe-on-rails/assets/64698693/fddbb145-fd28-4440-83cb-9a30ae7622cc)

![image](https://github.com/smunoz1988/recipe-on-rails/assets/64698693/9660c4e4-09ee-4b2f-9c6b-a9b8cf586951)

but as requested, We added it! 

![image](https://github.com/smunoz1988/recipe-on-rails/assets/64698693/e01a34e7-3466-4765-9411-d3cc0bbed332)

thanks for the suggestion.

Reply:
@smunoz1988 Hi, I know the wireframe doesn't have a button, I might have said to add it to the recipe details. Sorry for the misunderstanding.Anyways, it is nicely done! 🍾